group_vars/maintenance.yml

```diff
@@ -149,89 +149,110 @@ fsm_htcondor_enable: true
+    config:
+      - commands = ["{{ custom_telegraf_env }} /usr/bin/gxadmin iquery jobs-queued"]
+      - timeout = "15s"
+      - data_format = "influx"
+      - interval = "1m"
+  galaxy_jobs_queued_internal:
+    plugin: "exec"
+    config:
+      - commands = ["{{ custom_telegraf_env }} /usr/bin/gxadmin iquery jobs-queued-internal-by-handler"]
+      - timeout = "15s"
+      - data_format = "influx"
+      - interval = "1m"
+  galaxy_jobs_queue_overview:
+    plugin: "exec"
+    config:
+      - commands = ["{{ custom_telegraf_env }} /usr/bin/gxadmin iquery queue-overview --short-tool-id"]
+      - timeout = "30s"
+      - data_format = "influx"
+      - interval = "1m"
+  galaxy_jobs_destination_overview:
+    plugin: "exec"
+    config:
+      - commands = ["{{ custom_telegraf_env }} /usr/bin/gxadmin iquery queue --by destination"]
+      - timeout = "30s"
+      - data_format = "influx"
+      - interval = "1m
+  galaxy_oidc:
+    plugin: "exec"
+    config:
+      - commands = ["{{ custom_telegraf_env }} /usr/bin/gxadmin iquery users-with-oidc"]
+      - timeout = "15s"
+      - data_format = "influx"
+      - interval = "1m"
+  galaxy_workflow:
+    plugin: "exec"
+    config:
+      - commands = ["{{ custom_telegraf_env }} /usr/bin/gxadmin iquery workflow-invocation-status"]
+      - timeout = "15s"
+      - data_format = "influx"
+      - interval = "1m"
+  galaxy_workflow_totals:
+    plugin: "exec"
+    config:
+      - commands = ["{{ custom_telegraf_env }} /usr/bin/gxadmin iquery workflow-invocation-totals"]
+      - timeout = "15s"
+      - data_format = "influx"
+      - interval = "1m"
+  monitor_condor_queue:
+    plugin: "exec"
+    config:
+      - commands = ["sudo /usr/bin/monitor-condor-queue"]
+      - timeout = "10s"
+      - data_format = "influx"
+      - interval = "1m"
+  monitor_condor_util:
+    plugin: "exec"
+    config:
+      - commands = ["sudo /usr/bin/monitor-condor-utilisation"]
+      - timeout = "10s"
+      - data_format = "influx"
+      - interval = "1m"
+  postgres_extra:
+    plugin: "exec"
+    config:
+      - commands = [
+        "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-cache-hit",
+        "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-index-size",
+        "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-index-usage",
+        "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-table-bloat",
+        "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-table-size",
+        "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-unused-indexes",
+        "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-vacuum-stats",
+        "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-stat-bgwriter",
+        "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-stat-user-tables",
+        ]
+      - timeout = "60s"
+      - data_format = "influx"
+      - interval = "2m"
+
 
 # Role: hxr.monitor-cluster
 monitor_condor: true
-monitor_condor_split_util: true
```

Comment:
I assume we are removing this option because we do not have any panel that makes use of the per-group info right?

Reply:
The config does not exist on sn06 and we do not seem to monitor that. Also, I have integrated it into my command. It would need quite an amount of SQL work to parse the output from my command to build new panels for various use cases. I need to test and build a few panels and see if I can parse them all through SQL queries directly in the dashboard.

Reply:
I meant the server, not the playbook. The config variable exists in the playbook but telegraf conf file does not on the server.

We do not have to clean up the role (not sure if anyone other than us is referencing this role and using it), we just stop referencing it if we don't use it at the moment in our playbooks.

Reply:
> I meant the server, not the playbook. The config variable exists in the playbook but telegraf conf file does not on the server.

Weird, but then it is clear that we are not using it.

> not sure if anyone other than us is referencing this role and using it

I know that it seems like a strange suggestion at first precisely because of that. The thing is that this role is _local_ to this repository. It is not published on Ansible Galaxy. So we cannot possibly be breaking anybody else's setup, because if they are using it, it is because they forked our repository. If they keep their fork in sync with us, they are also keeping the removal of `monitor_condor_split_util: true` in sync.

Reply:
After building the required panels through the new command I will create a clean-up PR which will clean up the role and remove the stuff that are not in use.